starport/pkg/cosmosver/cosmosver.go

```diff
@@ -61,11 +64,14 @@ func (v Version) Major() MajorVersion {
 
 func (v Version) String() string {
 	switch v {
-	case StargateZeroFourtyAndAbove:
-		return "Stargate v0.40.0 (or later)"
+	case StargateZeroFortyToZeroFortyTwo:
+		return "Stargate v0.40.x - v0.42.x"
+
+	case StargateBelowZeroForty:
+		return "Stargate v0.39.9"
```

Comment:
This is not an existing sdk version.

But anyway what do you think about converting the `Version` type to a struct and storing the exact version number inside so we can just print it here?